satpy/utils.py

```diff
@@ -127,16 +116,14 @@ def logging_on(level=logging.WARNING):
 def logging_off():
     """Turn logging off.
     """
-    logging.getLogger('').handlers = [NullHandler()]
+    logging.getLogger('').handlers = [logging.NullHandler()]
 
 
 def get_logger(name):
-    """Return logger with null handle
-    """
-
+    """Return logger with null handler added if needed."""
     log = logging.getLogger(name)
     if not log.handlers:
-        log.addHandler(NullHandler())
+        log.addHandler(logging.NullHandler())
```

Comment:
While we're working with logging, can we add a line in the Issue template to switch on `debug_on` before reporting the issue? Maybe add a line about that in the doc too.

Reply:
Looks like github has a new templating system. We can use that and remove these templates. Maybe in a future PR?

https://help.github.com/articles/creating-issue-templates-for-your-repository/